src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const populationValues = [...document.querySelectorAll('.population')]
+  .map(element => {
+    const populationText = element.textContent;
+
+    try {
+      return parseInt(populationText.replace(/,/g, ''), 10);
+    } catch (error) {
+      return 0;
+    }
+  });
+
+const totalPopulation = populationValues
+  .reduce((total, population) => total + population, 0);
+const averagePopulation = totalPopulation / populationValues.length || 0;
```

Comment:
why do we need to have this `|| 0`? check this condition in console `1 / 0 || 1`. You will see output `Infinity`, not 1, because when we will divide something to `0` it's be `Infinity` and this `|| 0` redundant and never works